holidays/countries/trinidad_and_tobago.py

```diff
@@ -0,0 +1,197 @@
+    ObservedHolidayBase,
+    SUN_TO_NEXT_WORKDAY,
+    SAT_SUN_TO_NEXT_WORKDAY,
+    WORKDAY_TO_NEXT_WORKDAY,
+)
+
+
+class TrinidadAndTobago(
+    ObservedHolidayBase,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+):
+    """Trinidad and Tobago holidays.
+
+    References:
+        * <https://otp.tt/trinidad-and-tobago/national-holidays-and-awards/>
+        * <https://laws.gov.tt/ttdll-web2/revision/list?offset=380&q=&currentid=322>
+        * <https://www.guardian.co.tt/article-6.2.448640.3e9e366923>
+        * <https://www.guardian.co.tt/article-6.2.410311.3fe66fb00f>
+        * <https://www.nalis.gov.tt/resources/tt-content-guide/labour-day/>
+        * <https://www.facebook.com/groups/191766699268/posts/10160832951274269/>
+        * <https://www.facebook.com/100064996051675/posts/1101755938667598/?_rdr>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Trinidad_and_Tobago>
+        * <https://en.wikipedia.org/wiki/Trinidad_and_Tobago_Carnival>
+        * <https://en.wikipedia.org/wiki/Indian_Arrival_Day#Trinidad_and_Tobago>
+        * <https://www.timeanddate.com/holidays/trinidad/eid-al-fitr>
+        * <https://www.timeanddate.com/holidays/trinidad/diwali>
+    """
+
+    country = "TT"
+    default_language = "en_TT"
+    supported_categories = (OPTIONAL, PUBLIC)
+    # %s (estimated).
+    estimated_label = tr("%s (estimated)")
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observed, estimated)")
+    supported_languages = ("en_TT", "en_US")
+    # Trinidad and Tobago gained independence on August 31, 1962.
+    start_year = 1963
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=TrinidadAndTobagoHinduHolidays)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=TrinidadAndTobagoIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        super().__init__(*args, **kwargs)
+
+    def _populate_observed(self, dts: set[date], multiple: bool = False) -> None:
+        for dt in sorted(dts):
+            self._add_observed(
+                dt,
+                rule=WORKDAY_TO_NEXT_WORKDAY + SAT_SUN_TO_NEXT_WORKDAY
+                if len(self.get_list(dt)) > 1
+                else SUN_TO_NEXT_WORKDAY,
+            )
+
+    def _populate_public_holidays(self):
+        dts_observed = set()
+
+        # New Year's Day.
+        dts_observed.add(self._add_new_years_day(tr("New Year's Day")))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
```

Comment:
```suggestion
        dts_observed.add(self._add_good_friday(tr("Good Friday")))
```
Also Easter Monday and Corpus Christi, for completeness.